mountd/ipc.c

```diff
@@ -65,16 +65,13 @@ static int generic_event(int type, void *payload, size_t sz)
 
 static int parse_reload_configs(const char *pwddb, const char *smbconf)
 {
-	int ret;
-
 	pr_debug("Reload config\n");
 
 	usm_remove_all_users();
-	ret = cp_parse_pwddb(pwddb);
+	int ret = cp_parse_pwddb(pwddb);
```

Comment:
Doesn't this build warning? I think the original code is correct...

Reply:
@neheb,

In this codebase, declarations are always at the top of the block. Projects may choose this convention even when they don't limit themselves to ISO C90.

Reply:
@namjaejeon why would there be a warning?

Reply:
Yeah this is fixed.